pkg/collector/embed.go

```diff
@@ -19,6 +20,18 @@ func pySetup(paths ...string) (pythonVersion, pythonHome, pythonPath string) {
 	return py.PythonVersion, py.PythonHome, py.PythonPath
 }
 
+func pyPrepareEnv() error {
```

Comment:
I don't see this function being called from anywhere.

Also, if called from outside this file, should have a similar function in `embed_nopy.go`

Reply:
🤦‍♂️